### ISSUE TITLE
Custom action sample should not duplicate schemas 

### ISSUE THREAD

**Issue by carlosscastro** [microsoft/BotFramework-Composer#5942]
Currently, schema merge fails when the custom action sample is added. Change the csproj of sample custom actions in legacy runtime so that they include schemas only once. To do this, 2 things need to happen:

1) schemas are added in a single place

```xml
<ItemGroup>
    <Content Include="**/*.schema" />
    <Content Include="**/*.uischema" />
</ItemGroup>
```
2) In the propertygroup where TargetFramework is defined, add this to remove duplication

```xml
<ContentTargetFolders>content</ContentTargetFolders>
```

**Comment by yeze322** [comment]
@carlosscastro Carlos could you confirm if this issue was resolved after bf-cli released 4.12.0? I tried on my side and found it works.

**Comment by cwhitten** [comment]
Closing with the expectation that bf-cli 4.12 addresses this.
